spec/models/user_spec.rb

```diff
@@ -0,0 +1,8 @@
+describe User do
+  let (:user) {described_class.create(username: 'gwpmad',email: 'george@test.com',name: 'George',password: '12345')}
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Metrics/LineLength: Line is too long. [116/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)